lemarche/templates/layouts/base.html

```diff
@@ -17,11 +17,12 @@
     {# Use this to get the value of the CSRF token in JavaScript. #}
     <meta name="csrf-token" content="{{ csrf_token }}">
     <link rel="shortcut icon" href="{% static "favicon.ico" %}" type="image/ico">
+    {% bootstrap_css %}
+    {% bootstrap_javascript jquery='full' %}
     {% block extra_head %}{% endblock %}
 </head>
 <body>
-    <div class="layout">
-      {%block content%}{%endblock%}
-    </div>
+    {% bootstrap_messages %}
+    {%block content%}{%endblock%}
```

Comment:
Allez, je pinaille !

```suggestion
    {% block content %}{% endblock %}
```